packages/client/tests/functional/_utils/getTestSuiteInfo.ts

```diff
@@ -133,7 +133,7 @@ export function getTestSuiteSchema(suiteMeta: TestSuiteMeta, matrixOptions: Reco
 export function getTestSuiteMeta() {
   const testsDir = path.join(path.dirname(__dirname), '/')
   const testPath = expect.getState().testPath
-  const testRootDirName = testPath.replace(testsDir, '').split(path.sep)[0]
+  const testRootDirName = path.parse(testPath.replace(testsDir, '')).dir
```

Comment:
This allows us to support nested test suites, for example in the pr we have: 

 `findUnique/`
   1. `composite-embedded-list/`
       - test.ts
       - prisma/
       - .generated
    
   2. `composite-embedded-with-name/`
       - test.ts
       - prisma/
       - .generated

Reply:
So, am I undestanding this correctly, right now there are 2 options for nesting

```
/test-folder
   schema/prisma.schema
   test1.ts
   test2.ts --> both files will use the same schema
```

```
/test-folder
  subfolder1
      schema/prisma.schema
      test.ts
   subfolder2
      schema/prisma.schema
      test.ts -> both tests will use different schemas
```
?

If so, that might be worth documenting. Would you mind adding a few sentences on that to the testing docs?

Reply:
This is correct yes, It makes it easier to nest related tests, vs polluting the root test directory with X amount of directories. 

Sure I can add something in the docs.

Reply:
See: https://github.com/prisma/prisma/pull/14988